pages/components/VideoFooter.tsx

```diff
@@ -118,7 +118,7 @@ export const VideoFooter: React.FC<{
             </div>
           </div>
         </div>
-        <div id="videoFooterMenu" className="flex space-x-2 absolute">
+        <div id="videoFooterMenu" className="flex space-x-2 fixed">
```

Comment:
This is to allow the video footer component to stay in the same position even when there are enough blocks that mean that vertical scrolling is enabled